### PR TITLE
Update finders.js

### PR DESCRIPTION
**Pull request by XhmikosR**
Replace all backslash occurrences, not just the first one

### CODE REVIEW

File: src/utils/finders.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ function getDependencies(filename, basedir) {
   const localFilesToProcess = [filename]
 
   function handle(name, basedir, optionalDependencies) {
-    const moduleName = requirePackageName(name.replace(/\\/, '/'))
+    const moduleName = requirePackageName(name.replace(/\\/g, '/'))
```
Comment (XhmikosR) [inline src/utils/finders.js]: BTW if this was supposed to change only the first occurrence maybe it should be `'\'` and not a regex

**Comment by raeesbhatti** [comment]
Thanks for the PR @XhmikosR

**Comment by XhmikosR** [comment]
@raeesbhatti you guys should add lgtm.com to the repo after tightening linting :)

https://lgtm.com/projects/g/netlify/cli/?mode=list

**Comment by raeesbhatti** [comment]
Thanks for the suggestion @XhmikosR. Will definitely consider that

**Pull request merged**